doc/changelog/02-specification-language/15693-reverse-coercions.rst

```diff
@@ -0,0 +1,8 @@
+- **Added:**
+  Reversible coercions are coercions which cannot be
```

Comment:
Ideally, the changelog entry should link directly to the relevant documentation.